.github/workflows/upload-and-deploy.yaml

```diff
@@ -277,11 +278,30 @@ jobs:
         if: github.ref_name != 'main'
         run: echo "NAMESPACE=$GITHUB_REF_NAME" >> $GITHUB_ENV
 
-      - name: Copies over test files from ingestion bucket
+      - name: Fetch the most recent exports.
```

Comment:
Do we have ~2 weeks of exports from the develop input bucket to fetch? Or will this just get whatever is available even if it's not 2 weeks worth?

Reply:
Yeah, it acts like a list slice in Python `[:28]`.

Reply:
Nit: Gotcha, should we set this as an env var? so this way we can use it in both the two integration test for staging and development, and easier to change in the future.

Reply:
Ohh that's a good idea. I accidentally merged before reading this comment. I can fix as part of #117